olive/passes/onnx/optimum_merging.py

```diff
@@ -28,7 +28,15 @@ def is_accelerator_agnostic(accelerator_spec: AcceleratorSpec) -> bool:
 
     @staticmethod
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
-        return get_external_data_config()
+        config = {
+            "run_ort_postprocess": PassConfigParam(
```

Comment:
this seems be https://onnxruntime.ai/docs/performance/model-optimizations/graph-optimizations.html. 

Do we need other GraphOptimizationLevel values?

Reply:
Optimizations should be done the merging step. The goal of this postprocessing ORT step was only to run the most basic optimizations for the purpose of removing unused initializers, and even for that purpose it doesn't seem to work 100% with some models.